libraries/GCS_MAVLink/GCS_serial_control.cpp

```diff
@@ -169,7 +169,7 @@ void GCS_MAVLINK::handle_serial_control(const mavlink_message_t &msg)
         return;
     }
 
-    if (packet.flags & SERIAL_CONTROL_FLAG_BLOCKING) {
```

Comment:
i'd rather remove support for SERIAL_CONTROL_FLAG_BLOCKING, this looks like a dangerous piece of code, if the flow control pin is blocking the uart this would cause a reboot
i think we just remove the feature